packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -124,7 +124,7 @@ export class PersonalDetailsForm extends React.Component {
                 this.setState({ is_submit_success: false }, () => {
                     setSubmitting(false);
                 });
-            }, 3000);
+            }, 10000);
```

Comment:
Please update the timeout in `Financial Assessment` and other similar pages where this would be applicable

Reply:
Got it, Thank you 😊